package.json

```diff
@@ -33,9 +33,9 @@
     "js-lint": "eslint js/ && eslint --config js/tests/.eslintrc.json --env node build/ Gruntfile.js",
     "js-lint-docs": "eslint --config js/tests/.eslintrc.json assets/js/ sw.js",
     "js-compile": "npm-run-all --parallel js-compile-*",
-    "js-compile-standalone": "rollup --environment BUNDLE:false --config build/rollup.config.js",
-    "js-compile-bundle": "rollup --environment BUNDLE:true --config build/rollup.config.js",
-    "js-compile-plugins": "babel --no-babelrc js/src/ --out-dir js/dist/ --source-maps --presets=es2015 --plugins=transform-es2015-modules-strip",
+    "js-compile-standalone": "ROLLUP=true rollup --environment BUNDLE:false --config build/rollup.config.js",
+    "js-compile-bundle": "ROLLUP=true rollup --environment BUNDLE:true --config build/rollup.config.js",
+    "js-compile-plugins": "PLUGINS=true babel js/src/ --out-dir js/dist/ --source-maps",
```

Comment:
maybe don't want to do it this way (or use https://www.npmjs.com/package/cross-env), don't think we support the `--environment flag` but we could think about it

Reply:
I don't understand that change, `--no-babelrc` is it deprecated ?

Reply:
no it's because there isn't a way to pass options to a preset via cli